mountpoint-s3/src/inode.rs

```diff
@@ -97,6 +100,30 @@ impl Superblock {
         Self { inner: Arc::new(inner) }
     }
 
+    /// The kernel tells us when it removes a reference to an [InodeNo] from its internal caches via a forget call.
```

Comment:
It wasn't a priority for us - during alpha, it did not feel essential to cleanup the superblock list of Inodes.

Caching doesn't come into why we're implementing forget I don't think, but it may (IDK) influence _when_ the kernel is telling us its forgotten about a particular Inode.